### PR TITLE
docs(CONTRIBUTING.md): fix some typos

### PR DESCRIPTION
**Pull request by RS-labhub**
Fixed typos and grammatical errors

### Checklist

- [ ] The Pull Request has tests
- [ ] A changelog file has been created under `changelog/unreleased/kong` or `skip-changelog` label added on PR if changelog is unnecessary. [README.md](https://github.com/Kong/gateway-changelog/README.md)
- [x] There is a user-facing docs PR against https://github.com/Kong/docs.konghq.com - PUT DOCS PR HERE

### Issue reference

Fix #_[issue number]_

### CODE REVIEW

**Comment by AndyZhang0707** [comment]
Hi @lena-larionova could you please take a look at this PR? thanks

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -207,7 +207,7 @@ makes it significantly easier to use or understand Kong, congratulations!
 You are now an official contributor to Kong. Get in touch with us to receive
 your very own [Contributor Badge](#contributor-badge)!
 
-Your change will be included in the subsequent release Changelog, and we will
+Your change will be included in the subsequent release of Changelog, and we will
```
Comment (lena-larionova) [inline CONTRIBUTING.md:210]:
```suggestion
Your change will be included in the subsequent release and its changelog, and we will
```

**Review by lena-larionova — COMMENTED** [review]
One minor edit, otherwise lgtm

**Comment by RS-labhub** [comment]
@lena-larionova I fixed it from my side. Could you please check it from you side?

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -415,11 +415,11 @@ suite.
   output
 - `spec/03-plugins` contains tests (both unit and integration) for the bundled
   plugins (those plugins still live in the core repository as of now, but will
-  eventually be externalized)
+  eventually, be externalized)
```
Comment (lena-larionova) [inline CONTRIBUTING.md:418]:
```suggestion
  eventually be externalized.)
```
This comma isn't necessary here.

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -338,7 +338,7 @@ written so that:
 
 The body of your commit message should contain a detailed description of your
 changes. Ideally, if the change is significant, you should explain its
-motivation, the chosen implementation, and justify it.
+motivation, and the chosen implementation, and justify it.
```
Comment (lena-larionova) [inline CONTRIBUTING.md:341]:
```suggestion
motivation and the chosen implementation, and justify it.
```
There's an extra comma here.

**Review by lena-larionova — COMMENTED** [review]
Couple of comma edits, for some reason the whole file didn't load for me last time 🤔 .

**Comment by RS-labhub** [comment]
@lena-larionova thanks for mentioning.

**Review by lena-larionova — APPROVED** [review]

**Comment by RS-labhub** [comment]
@lena-larionova could you ping the 2nd approver, so that it could be merged?

**Review by jschmid1 — APPROVED** [review]

**Comment by RS-labhub** [comment]
@AndyZhang0707 all checks have passed, could you please merge it?

**Pull request merged**